Sources/ArcGISToolkit/Components/Form/FormInputs/MultiLineTextInput.swift

```diff
@@ -128,4 +103,44 @@ private extension MultiLineTextInput {
     var fieldType: FieldType {
         model.featureForm!.feature.table!.field(named: element.fieldName)!.type!
     }
+    
+    /// The body of the text input when the element is editable.
+    var textEditor: some View {
+        HStack(alignment: .bottom) {
+            TextEditor(text: $text)
+                .scrollContentBackgroundHidden()
+            if isFocused && !text.isEmpty && inputModel.isEditable {
+                ClearButton { text.removeAll() }
+            }
+        }
+        .background(.clear)
+        .focused($isFocused)
+        .foregroundColor(isPlaceholder ? .secondary : .primary)
+        .frame(minHeight: 75, maxHeight: 150)
```

Comment:
Will this prevent the height from expanding to display all the text? In the issue we talked about always showing the entirety of the text, or am I misremembering?

Reply:
Yes, my bad I see that you did mention it in 206. I think I have a solution but it will depend on the `FormViewModel.focusedElement` proposed in #483 in order to be usable so I'll take this back to draft in the mean time to get that merged first.